.pre-commit-config.yaml

```diff
@@ -16,7 +16,7 @@ repos:
       - id: check-executables-have-shebangs # Ensures that (non-binary) executables have a shebang.
       - id: check-json # This hook checks json files for parseable syntax.
       - id: check-shebang-scripts-are-executable # Ensures that (non-binary) files with a shebang are executable.
-      - id: pretty-format-json # This hook sets a standard for formatting JSON files.
+# Seems unnecessary:      - id: pretty-format-json # This hook sets a standard for formatting JSON files.
```

Comment:
Why remove this? Is it breaking something?

Reply:
It was failing on the `frag_72.json` file, but I'm not sure why. JSON files aren't a central part of this code base so I thought maybe this was unnecessary.